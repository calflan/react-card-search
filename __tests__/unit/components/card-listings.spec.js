import React from 'react';
import CardListings from '../../../src/js/components/card-listings';
import Card from '../../../src/js/components/card';

describe('Contact Component', () => {
  let component;

  beforeEach(() => {
    component = shallow(<CardListings />);
  });

  test('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render Card component when status is set to complete in state', () => {
    component.setState({status: "complete"});

    const card = component.find(Card);

    expect(card).toBeDefined();
  });
});