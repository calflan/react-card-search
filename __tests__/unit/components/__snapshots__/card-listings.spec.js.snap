// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component should match snapshot 1`] = `
<Fragment>
  <Header />
  <section
    className="container"
    id="card-container"
  >
    <h2>
      Our New Job cards
    </h2>
    <ul
      class="card-listings"
    >
      <loader />
    </ul>
    <a
      className="scroll-to-top"
      href="#card-container"
    >
      â‡§
    </a>
  </section>
</Fragment>
`;
